src/components/simpleDropdown/__tests__/simpleDropdown.test.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { shallowComponent } from '../../../../config/jest.setupTests';
-import { SimpleDropdown } from '../simpleDropdown';
+import { SimpleDropdown, SimpleDropdownItemProps } from '../simpleDropdown';
```

Comment:
Removed in 979fe3812afac4434e397acb3cc584e40da9debc .

That was leftover from earlier iteration, where instead of having type with fields, I tried using tuple. But then `test:types` complained and I had to tell the type explicitly there. Later I switched to interface with named fields, but left this hint around.

Looks like now typescript can infer type correctly.